portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,142 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <meta charset="UTF-8" />
+    <link rel="shortcut icon" href="assets/cartoon-edited.png" />
+    <link rel="stylesheet" type="text/css" href="styles/styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Crimson+Text|Work+Sans:400,700"
+      rel="stylesheet"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Aradhya's Website" />
+    <meta name="theme-color" content="#000000" />
+    <title>Who is Aradhya Bansal?</title>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+    <main>
+      <div class="content">
+        <h1 class="title">aradhya ▪<em>noun</em></h1>
+          <span>
+            <h3 class="title" style="font-size: 2em; display: inline;">[<em>Say</em> uh-rah-dee-uh]</h3>
+            <img class="icon" src="assets/sound.png" onclick="playAudio('assets/name.mp3')">
+          </span>
+        <h5 class="subtitle">
+          <em>
+            1. software engineering and business student enrolled at western
+            university 2. likes to code and work on startups. 3. passionate
+            about product development, business strategy, basketball, and
+            self-improvement. 4. dreams big and gets s#$% done
+          </em>
+        </h5>
+        <p class="main">
+          <em>"Vulnerability doesn't come after trust - it precedes it."</em> ~
+          <a
+            class="list-link"
+            href="https://fourminutebooks.com/the-culture-code-summary/"
+            target="#"
+            ><u>The Culture Code:</u></a
```

Comment:
Nit: I would out this end bracket on the same line as above and start and finish the </a> on a new line (here and applies to similar blocks below)

Reply:
Got it, I think a lot of these earlier commits were pretty spaghetti so I apologize for that.